ravenpy/config/rvs.py

```diff
@@ -250,7 +250,7 @@ class RVI(RV):
     # come after `:SoilModel`
     _post_tmpl = """
     :EvaluationMetrics     {evaluation_metrics}
-    {evaluation_periods}
+    :EvaluationPeriod      {evaluation_periods}
```

Comment:
Unless there is a problem with the original formulation, this should not be changed. 

This is the definition of the EvaluationPeriod command:
 ```
@dataclass
class EvaluationPeriod(RavenCommand):
    """:EvaluationPeriod [period_name] [start yyyy-mm-dd] [end yyyy-mm-dd]"""

    name: str
    start: dt.date
    end: dt.date

    template = ":EvaluationPeriod {name} {start} {end}"

    def to_rv(self):
        return self.template.format(**asdict(self))
```
In practice, use something like (untested):
```
model.config.rvt.evaluation_periods = [EvaluationPeriod('wet', dt,date(1990, 1, 1), dt.date(2000, 1, 1))]
```

Reply:
What do you mean by "original formulation" ? 
https://github.com/CSHS-CWRA/RavenPy/blob/427f7caf0562928db1951e42b1217727bf925c62/ravenpy/config/commands.py#L80

If there's a but with the EvaluationPeriod configuration, let's fix it in another PR.